targets/XPlatCppSdk/source/LibUnitTest/PlayFabEntityTest.cpp

```diff
@@ -216,6 +216,7 @@ namespace UnittestRunner
         }
         static void SetObjectsCallback(const SetObjectsResponse& result, void*)
```

Comment:
Don't do the UNREFERENCED_VARIABLE(result); thing below.
Fix result here by removing the result keyword.

Reply:
So literally:
static void SetObjectsCallback(const SetObjectsResponse&, void*)
That's correct syntax and it'll remove that warning/error.